02 - Community Expansions/02.01 Plugins by Category/Plugins for collaboration.md

```diff
@@ -0,0 +1,27 @@
+---
+aliases:
+- Collaboration plugins
+- Plugins for collaborative editing
+tags: 
+- seedling 
+publish: true
+---
+
+
+# Plugins for collaboration
+
+%% Add a description below this line. It doesn't need to be long: one or two sentences should be a good start. %%
+If you ever had to work on some kind of group project and wanted to collaborate with your peers, you may have wondered how you could achieve this right inside your favourite Markdown editor ([[Obsidian]], obviously). Whether you just want to keep track of external notes inside your vault or co-edit notes in real-time, this is where you find the right tools.
+
+## Plugins in this category
+
+- [[02 - Community Expansions/02.05 All Community Expansions/Plugins/sekund|Sekund]] is social media for your notes. Engage others in peer-to-peer thinking and get feedback on drafts. Start discussions and gather in your group chat.
+- [[obsidian-git|Obsidian Git]] can help you keep track of external changes to your notes by leveraging the [version control](https://www.wikiwand.com/en/Version_control) system [Git](https://book.git-scm.com/). Vaults could be shared through [GitHub](https://docs.github.com/en/get-started/quickstart).
+- [@egradman's Etherpad Lite plugin](https://github.com/egradman/obsidian-etherpad-lite)(currently still in [[Plugins in Beta|beta]]) lets you upload any note to a self-hosted [Etherpad Lite](https://etherpad.org/) server and share the URL for live collaborative editing. This lightweight tool links your local document to its online version via a YAML key and continuously updates it as you type.
+- [[for Plugin Developers|Plugin developers]] might also be interested in [[netwik|Netwik]] which wanted to provide global access to notes by hosting them on a specified remote server. Be aware that this plugin is currently [[Discontinued plugins|unmaintained]].
+
+## Related categories
+
+%% Add links to other 02.02 - Category notes %%
+
+[[Plugins for Editing Notes]]
```

Comment:
Looking at this section in other categories, they have the related categories in a bullet list. I think this helps maintenance and comprehension... (It's OK to have a bullet list with 1 item in 😄 )

Reply:
Thanks for noticing, I changed it for consistency!